TutByParametrizedTests/Task2tests/Task2tests/Pages/SeleniumeasyPages/AlertBoxDemoPage.cs

```diff
@@ -0,0 +1,116 @@
+﻿using OpenQA.Selenium;
+using Training.Pages;
+
+namespace Task2tests
+{
+    public class AlertBoxDemoPage : MainPage
+    {
+        private const string alertBoxDemoPageUrl = "https://www.seleniumeasy.com/test/javascript-alert-box-demo.html";
+        private readonly By firstAlertBoxButton = By.CssSelector("button[onclick='myAlertFunction()']");
+        private readonly By confirmBoxButton = By.CssSelector("button[onclick='myConfirmFunction()']");
+        private readonly By promtBoxButton = By.CssSelector("button[onclick='myPromptFunction()']");
+        private readonly By confirmIDField = By.CssSelector("#confirm-demo");
+        private readonly By promptIDField = By.CssSelector("#prompt-demo");
```

Comment:
Please, add for these private properties prefix _ (for example, _firstAlertBoxButton)